llama_stack/providers/inline/vector_io/faiss/faiss.py

```diff
@@ -286,17 +287,20 @@ async def _update_openai_vector_store(self, store_id: str, store_info: dict[str,
         assert self.kvstore is not None
         key = f"{OPENAI_VECTOR_STORES_PREFIX}{store_id}"
         await self.kvstore.set(key=key, value=json.dumps(store_info))
+        self.openai_vector_stores[store_id] = store_info
 
     async def _delete_openai_vector_store_from_storage(self, store_id: str) -> None:
         """Delete vector store metadata from kvstore."""
         assert self.kvstore is not None
         key = f"{OPENAI_VECTOR_STORES_PREFIX}{store_id}"
         await self.kvstore.delete(key)
+        if store_id in self.openai_vector_stores:
+            del self.openai_vector_stores[store_id]
 
     async def _save_openai_vector_store_file(
         self, store_id: str, file_id: str, file_info: dict[str, Any], file_contents: list[dict[str, Any]]
     ) -> None:
-        """Save vector store file metadata to kvstore."""
+        """Save vector store file data to kvstore."""
```

Comment:
Isn’t this metadata?

Reply:
Two keys are being stored here, one is 
`openai_vector_stores_files:v3::my-docs:file-<fileid>`
and the other is
`openai_vector_stores_files_contents:v3::my-docs:file-<fileid>`

I had changed the docstring in "_delete_openai_vector_store_file_from_storage" to refer to "data" rather the "file metadata" as it now removed both keys (as I saw it contents and metadata)
so I also changed the docstring here for consistentcy as it also deals with both keys